docs/variants/clevo_nv41/fwupd_usage.md

```diff
@@ -120,7 +120,9 @@ $ sudo ninja -C build install
 
 ## Updating firmware from LVFS
 
-1. Use the following command to update firmware with fwupd:
+1. Disable Secure Boot.
```

Comment:
@IgorBagnucki Can we elaborate how to do that? Or link to a document where we already have explained that?

Reply:
Also, the lockdown mode may be a bit different depending on a distro used.

How about also disabling lockdown as suggested here: https://github.com/flashrom/flashrom/issues/59 ?

Reply:
We don't have a specific procedure for disabling Secure Boot, closest we have is https://docs.dasharo.com/unified-test-documentation/dasharo-security/206-secure-boot/ which describes *enabling* Secure Boot, but the process is almost identical.

For disabling kernel lockdown, as far as i can tell the sysrq+x disable was never accepted upstream and only some kernels had it for some time, but dropped it due to security problems.

Reply:
@IgorBagnucki Let's link it then with the releveant description and then we can merge

Reply:
@macpijan The link is added.